app/build.gradle.kts

```
@@ -42,6 +42,8 @@ android {
 }
 
 dependencies {
+    implementation(project(mapOf("path" to ":domain")))
```

app 모듈은 domain 모듈에 의존하도록 잘 해주셨네요!
```suggestion
    implementation(project(":domain"))
```
다만 하나의 모듈만 정의할 시 꼭 map으로 넘기지 않아도 됩니다 🙂